src/View/Helper/TimeHelper.php

```diff
@@ -33,6 +34,17 @@ class TimeHelper extends Helper
     use StringTemplateTrait;
 
     /**
+     * Default config for this class
+     *
+     * @var array
+     */
+    protected $_defaultConfig = [
+        'defaultLocale' => null,
+        'defaultFormat' => null,
+        'defaultOutputTimezone' => null,
+    ];
```

Comment:
Agreed. I am not going to do TimeHelper support in the upcoming PR unless someone has a good reason.